### PR TITLE
Added Xdebug support to docker

### PR DESCRIPTION
**Pull request by JonaMata**

### CODE REVIEW

**Review by Jessseee — APPROVED** [review]
LGTM. Really needed that.

File: .env.docker.example [hunk 1]
```diff
@@ -5,6 +5,8 @@ FALLBACK_COOKIE_DOMAIN=localhost
 
 DEBUG=true
 
+PHP_IDE_CONFIG=serverName=saproto-debug
```
Comment (Jessseee) [inline .env.docker.example:8]: Does this take in to account the docker name variable?

**Review by Jessseee — CHANGES_REQUESTED** [review]

File: .env.docker.example (hunk 1 above)
Comment (JonaMata) [reply .env.docker.example:8]: No, this is the name you give the debug server in your IDE, you just have to make sure that the two match

Comment (Jessseee) [reply .env.docker.example:8]: Maybe we should add that to the README?

Comment (JonaMata) [reply .env.docker.example:8]: wow, so smart

**Review by Jessseee — APPROVED** [review]
LGTM

**Pull request merged**